### PR TITLE
initiate pull request for data work in progress

### PR DESCRIPTION
**Pull request by claremarie**

### CODE REVIEW

**Comment by enki-hq** [comment]
Invalid headline on line 0: stub

File: data/exploratory-data-analysis/summarize-represent-and-interpret-data-in-multiple-categorial-and-quantitative-variables/readme.md [hunk 1]
```diff
@@ -0,0 +1 @@
+stub
```
Comment (lizTheDeveloper) [inline data/exploratory-data-analysis/summarize-represent-and-interpret-data-in-multiple-categorial-and-quantitative-variables/readme.md]: You want these to have all the metadata, plus `stub: true`

File: data/exploratory-data-analysis/summarize-represent-and-interpret-data-on-a-single-count-or-measurement-variable/mean-median-mode.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+category: fundamental
+
+tags:
+
+  - central-tendency
+  - statistics
+
+notes: 'this is my first attempt, so bear with me'
+
+
```
Comment (lizTheDeveloper) [inline data/exploratory-data-analysis/summarize-represent-and-interpret-data-on-a-single-count-or-measurement-variable/mean-median-mode.md]: here's where you'd put `stub: true`

File: data/exploratory-data-analysis/summarize-represent-and-interpret-data-on-a-single-count-or-measurement-variable/mean-median-mode.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+## Content
+
+One of the first questions you should ask when encountering numerical data is: "What is a typical value for this dataset?"
+
+
+---
+---
+## Practice
+
+Considering the list of integers
```
Comment (lizTheDeveloper) [inline data/exploratory-data-analysis/summarize-represent-and-interpret-data-on-a-single-count-or-measurement-variable/mean-median-mode.md]: This question is great, perfect distinction between practice and revision question

**Comment by enki-hq** [comment]
Invalid headline on line 0: stub

**Comment by enki-hq** [comment]
Invalid section title on line 26: ---

**Comment by enki-hq** [comment]
Invalid section title on line 27: ---

**Comment by enki-hq** [comment]
Invalid section title on line 27: ---

**Comment by enki-hq** [comment]
Invalid section title on line 27: ---

**Pull request merged**